src/gui/static/src/app/components/pages/wallets/wallet-detail/wallet-detail.component.html

```diff
@@ -3,7 +3,10 @@
     <div class="id-column">{{ i + 1 }}</div>
     <div class="address-column">
       <img src="../../../../../assets/img/qr-code-black.png" (click)="showQrCode($event, address.address)">
-      <p>{{ address.address }}</p>
+      <p class="click-to-copy" [ngClass]="{ copying: address.copying }"
+         (click)="copyAddress($event, address)" (mouseleave)="address.copying = false">
+        {{ address.address }} <span data-label="Copied" class="copy-label">Copy</span>
```

Comment:
Please, use ngx-translate for the button text and the animated confirmation text.